recipes/gps-timemachine/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "gps-timemachine" %}
+{% set version = "1.1.5" %}
+{% set sha256 = "7758810c5417e2f7a80834d422d01f55f05709897bd06df107f19d2f8c896fa7" %}
+{% set build_number = 0 %}
+
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gps_timemachine-{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+build:
+  number: "{{ build_number }}"
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  noarch: python
+
+requirements:
+  host:
+    - python ~=3.10
```

Is this a Python 3.10 only recipe? Then please use the following
```suggestion
    - python 3.10
```

---

Thanks! It should be marked >=3.10 :)